tests/tests/test_fields.py

```diff
@@ -0,0 +1,52 @@
+from django.test import TestCase
+
+from gdpr.fields import Fields
+from tests.anonymizers import CustomerAnonymizer
+from tests.models import Customer
+
+LOCAL_FIELDS = ("first_name", "last_name")
+
+BASIC_FIELDS = (
+    "primary_email_address",
+    ("emails", (
+        "email",
+    )),
+)
+
+MULTILEVEL_FIELDS = (
+    ("accounts", (
+        "number",
+        "owner",
+        ("payments", (
+            "value",
+            "date"
+        ))
+    )),
+)
+
+
+class TestFields(TestCase):
```

Comment:
lepší názvy testů.. moc to neříká co to dělá

Reply:
Tohle má testovat classu Fields

Reply:
myslím názvy metod v testech. test_local_all mimoc neřekne co to dělá..